jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-helpers.c

```diff
@@ -20,6 +20,7 @@
 #include "ecma-builtins.h"
 #include "ecma-gc.h"
 #include "ecma-objects.h"
+#include "ecma-exceptions.h"
```

Comment:
should not include

Reply:
Yeah it's unnecessary. But please for the lord use whole sentences and punctuation marks rather than halfwords.